### ISSUE TITLE
Is it possible to force the response to fill or re-indent to a specific width?

### ISSUE THREAD

**Issue by oatmealm** [daedsidog/evedel#12]
Would be helpful to have the `output` not spread over multiple lines. The overlay does not seem to respond to most re-alignment command except word-wrap...

**Comment by daedsidog** [comment]
I'm not sure I understand what you mean. Are you referring to the indentation formatting that occurs after the response is processed?

**Comment by oatmealm** [comment]
Yes. The repose echoed into the overlay when processing the directives. Right now it just flows and wraps... so it's not very readable. I thought it might help if it could fill to a specific default column, preferably the buffer's own default?

**Comment by daedsidog** [comment]
What is the major mode of the buffer you're trying to use it in? Right now, it just uses the default indentation functions:

```
(unless (eq indent-line-function #'indent-relative)
                      (indent-region beg end))
```

I planned to export the processing function anyways, but I'm still curious as to why it wraps text for you, as it shouldn't do that.

Unless, I am still misunderstanding you, and you mean the actual text of the directive, which is part of the overlay itself, and not the text surrounded by the overlay. But I don't think it's that, since that text uses the buffer's default column width anyway...
